1-reql-language/geo-support/ruby.md

```diff
@@ -107,7 +107,7 @@ While there *is* a [circle] command, it approximates a circle by defining either
 
 # Geospatial indexes #
 
-To create indexes on fields containing geometry objects, you simply use the standard [index_create](/api/ruby/index_create/) command, setting the `geo` optional argument to `Prue`. In JavaScript, this would be:
+To create indexes on fields containing geometry objects, you simply use the standard [index_create](/api/ruby/index_create/) command, setting the `geo` optional argument to `True`. In Ruby, this would be:
```

Comment:
This should also probably be a lower-case `true`?
(We can also take care of that on our end after merge.)